packages/@aws-cdk/aws-certificatemanager/lib/certificate.ts

```diff
@@ -247,6 +261,8 @@ export class Certificate extends CertificateBase implements ICertificate {
       certificateTransparencyLoggingPreference,
     });
 
+    Tags.of(cert).add(NAME_TAG, props.certificateName || this.node.path);
```

Comment:
The Tag value has a limit of 255 characters.  Can you add something here to restrict the value to that? I don't think it is as much of a problem if the user provides a `certificateName`, but `node.path` _might_ hit that limit.

Maybe just a `value.slice(0, 255);`

https://docs.aws.amazon.com/acm/latest/userguide/tags-restrictions.html

Reply:
Nice catch; I have sliced the auto-generated certificate name but left the user inputted value alone.